docs/marionette.region.md

```diff
@@ -56,6 +56,8 @@ var App = Mn.Application.extend({
 });
 ```
 
+[Live example](http://jsfiddle.net/charlyjazz/gze5ykov/6/)
```

Comment:
this looks good.  would you be up for swapping the router to Backbone.Router?
AppRouter will be available as a separate lib for v4, so rather than do this now and update it in a few weeks, using Backbone's router would keep this example solid for v4 as well.

Reply:
I will work on that and I will notify you

Reply:
Ready. I have updated the fiddle. Check it